blogs/nlp/LOREN/README.md

```diff
@@ -0,0 +1,135 @@
+
+1. A claim is SUP iff all phrases are SUP
+2. A claim is REF iff there's at least one phrase that is REF
+3. A claim is NEI iff neither above.
+
+Following the above logic rules, loren further soften these rules into a probability distribution.
+
+1. $q^{T}_{\phi}(y_z = SUP) = \sum_{i=1}^{|z|}q_{\phi}(z = SUP)$
+2. $q^{T}_{\phi}(y_z = REF) = 1- \sum_{i=1}^{|z|}(1-q_{\phi}(z = SUP))$
+3. $q^{T}_{\phi}(y_z = NEI) = 1-q^{T}_{\phi}(y_z = SUP)-q^{T}_{\phi}(y_z = REF)$
+
+where
+
+1. $y\in{SUP,REF,NEI}$, which is the veracity of a claim.
+2. $z_i\in{SUP,REF,NEI}$, which is the latent variable represents the veracity of a phrase.
+3. $y_z$, which is the logic aggregation of the latent variable $z$, meaning that $y_z$ is a categorical distribution over three labels ${SUP,REF,NEI}$.
+
+The above aggregation of the soft logic serves to distill the knowldge to $p_{\theta}(y|z,x)$ (Probability of the claim label given the phrase-level label and input x),
+which is the probability we eventually want to optimize.
+
+## Experiment
+
+The model is trained on BERT and RoBERTa, and eventually it is evaluated on the dataset FEVER 1.0 shared task that is split into training, development and blind test set.
+Before we analyze the experiment result, I will summarize the experiment metrics they used to evaluate the performance[2]:
+
+### Experiment metrics for the performance of the model:
+
+1. Label Accuracy (LA): The accuracy of predicted label for claim regardless of retrieved evidence.
+2. FEVER score (FEV): The accuracy of both predicted label and retrieved evidence.
+
+![image2](./result1.png)
+
+Let's look at the above overall performance of LOREN. We can see from the result that LOREN is either outperforming or comparable with other
+baseline method except for LisT5. This is mainly because the PLM for LisT5, which is T5, is larger than BERT and RoBERTa. However, LOREN still outperforms LisT5 on the FEVER score of the development set (dev). To summarize, the performance of LOREN is decent and it has the potential to achieve a better result.
+
+![image3](./result2.png)
+In addition, we will dive into the most important part of the experiment: a case study of the LOREN's interpretability.
+For claim one in the above figure, LOREN can detect which phrase is the culprit in the claim, and it's also providing the reasoning that why that phrase is causing the mistake, i.e., replacing the false phrase "the number three" with "the number one". In the third claim, LOREN shows its ability to detect multiple culprit phrases in the claim. However, for the second claim, the predicted result should be NEI. LOREN made this mistake mainly because of the evidence retrieval module and MRC module. First, the evidence retrieval module did not draw the correct evidence, and then the MRC module still replaced the original phrase even though there's no direct evidence given. Here's another example that indicates a drawback of LOREN:
+
+![image4](./result3.png)
```

Comment:
more example with interesting claims?

Reply:
Sure, would it be good if I screen-shot the result from the demo as an illustration?